### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Dadadon**
- Created the home page for mobile.
- Created the about page for mobile.
- Featured speakers and main program section loads data dynamically
- Users can navigate to desired page or section of the section of the active page using navbar
- All pages are fully responsive
- Implemented the user interactions: link, mobile menu.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -31,44 +31,54 @@
...
+          <a href="#time-cards" onclick="closeNav()">Timeline</a>
+          <a href="#partner" onclick="closeNav()">Partner</a>
+          <a href="index.html" onclick="closeNav()">Home</a>
+          <button class="h-50 align-self-center secondary head-btn">AI
+            Campaign</button>
+        </nav>
+      </div>
+
     </nav>
   </header>
```
Comment (ishimwezachee) [inline about.html]: - [ ]  Your navigation should be divided into top dark bar and bottom white bar as provided from Behance design 

Figma design 
image
Behance design 
image

File: about.html [hunk 2]
```diff
@@ -31,44 +31,54 @@
...
   </section>
   <!-- logo section -->
-  <section class="logo-container bg-white p-5" id="ai-logo" >
+  <section class="logo-container bg-white p-5" id="ai-logo">
     <h3 class="featured-header text-center">Angel Investor Business Summit 2022 Logo</h3>
     <p class="divider text-center">_________</p>
-    <p class="mb-5 text-center">The logo of Angel Investor Business Summit 2022 was decided through the logo competition that was held on May 5th
+    <p class="mb-5 text-center">The logo of Angel Investor Business Summit 2022 was decided through the logo competition
+      that was held on May 5th
     </p>
```
Comment (ishimwezachee) [inline about.html]: - [ ] Your background should be images with a certain background color on it ,
Your design 
image

Behance design 

image

File: index.html [hunk 3]
```diff
@@ -30,43 +30,68 @@
...
+  <section class="header-container header-background p-3 w-100 pt-md-5 ps-md-5 pe-md-5 pb-md-5">
+    <h3 class="mt-md-5 ms-md-5 ps-md-5 me-md-5 pe-md-5">"Hello! Entrepreneurs"</h3>
+    <h1 class="ms-md-5 ps-md-5 me-md-5 pe-md-5">Angel Investor Business Summit 2022</h1>
+    <p class="ms-md-5 ps-md-5 me-md-5 pe-md-5 pt-md-5 pb-md-5">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloribus, iure officiis? Eveniet eos nulla, unde
+      voluptatum officiis animi rerum! Rem, quos illum ratione incidunt reiciendis obcaecati ad ipsa eius quisquam?
+    </p>
+    <h2 class="ms-md-5 ps-md-5">2022.12.25(SUN) ~ 27(TUE)</h2>
+    <h6 class="ms-md-5 ps-md-5">@ Jamaica Conference Center, Conference Room 1</h6>
   </section>
   <!-- Main program -->
```
Comment (ishimwezachee) [inline index.html]: - [ ]  Follow the Provided Behance design fonts, color, and margin.  Please resemble the provided design 

Your design 
image

Behance design 
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Dadadon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well described 👍 

## Required Changes ♻️
You should Provide a short video in your Readme documentation that summarizes what you've done in your Project based on provided guidelines from the requirements 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (DeliceLydia) [reply about.html]: Addressed

**Review by DeliceLydia — APPROVED** [review]
## Hi @Dadadon, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**